src/object.c

```diff
@@ -468,12 +847,12 @@ int paint_path_entry(int flags, int ebx, int ecx, int edx, rct_drawpixelinfo* dp
 		{
 			//Draws preview for scenario editor!
 			int b = ((uint32_t*)(ebp + 2))[0];
-			b += 0x47;
-			ecx -= 0x31;
-			edx -= 0x11;
+			b += 71;
+			ecx -= 49;
+			edx -= 17;
 			gfx_draw_sprite(dpi, b, ecx, edx, ebp);
```

Comment:
After you have written the code and tested it works, you can go back and refactor variable names like these from registers to more meaningful names like e.g. spriteIndex, x, y and colour.